data/src/main/java/com/linkedin/data/codec/AbstractJacksonDataCodec.java

```diff
@@ -296,6 +296,12 @@ public void startMap(DataMap map) throws IOException
       _generator.writeStartObject();
     }
 
+    @Override
+    public Iterable<Map.Entry<String, Object>> orderMap(DataMap map)
```

Comment:
This is not a good idea. The jackson codec is used for serializing data on the servers and making this change here will affect performance of services.

Reply:
Update code ,can you please check? Now I limit the scope of the change so it only applies when writing properties.